src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,119 @@
+                {
+                    UserAuthenticationModel postUserValue = new UserAuthenticationModel() { EncryptedTicket = Request.Cookies[_generalSettings.GetSBLCookieName] };
+                    HttpResponseMessage response = await client.PostAsync(endpointUrl, new StringContent(postUserValue.ToString(), Encoding.UTF8, "application/json"));
+                    if (response.StatusCode == HttpStatusCode.OK)
+                    {
+                        Stream stream = await response.Content.ReadAsStreamAsync();
+                        userAuthentication = serializer.ReadObject(stream) as UserAuthenticationModel;
+                        if (userAuthentication.IsAuthenticated)
+                        {
+                            List<Claim> claims = new List<Claim>();
+                            string issuer = _generalSettings.GetPlatformEndpoint;
+                            claims.Add(new Claim(AltinnCoreClaimTypes.UserId, userAuthentication.UserID.ToString(), ClaimValueTypes.String, issuer));
+                            claims.Add(new Claim(AltinnCoreClaimTypes.UserName, userAuthentication.Username, ClaimValueTypes.String, issuer));
+                            claims.Add(new Claim(AltinnCoreClaimTypes.PartyID, userAuthentication.PartyID.ToString(), ClaimValueTypes.Integer32, issuer));
+                            claims.Add(new Claim(AltinnCoreClaimTypes.AuthenticateMethod, userAuthentication.AuthenticationMethod.ToString(), ClaimValueTypes.String, issuer));
+                            claims.Add(new Claim(AltinnCoreClaimTypes.AuthenticationLevel, userAuthentication.AuthenticationLevel.ToString(), ClaimValueTypes.Integer32, issuer));
+                            if (userAuthentication.SSN != null)
+                            {
+                                claims.Add(new Claim(AltinnCoreClaimTypes.SSN, userAuthentication.SSN, ClaimValueTypes.String, issuer));
+                            }
+
+                            ClaimsIdentity identity = new ClaimsIdentity(_generalSettings.GetClaimsIdentity);
+                            identity.AddClaims(claims);
+                            ClaimsPrincipal principal = new ClaimsPrincipal(identity);
+                            await HttpContext.SignInAsync(
+                                JwtCookieDefaults.AuthenticationScheme,
+                                principal,
+                                new AuthenticationProperties
+                                {
+                                    ExpiresUtc = DateTime.UtcNow.AddMinutes(int.Parse(_generalSettings.GetJwtCookieValidityTime)),
+                                    IsPersistent = false,
+                                    AllowRefresh = false,
+                                });
+
+                            if (userAuthentication.TicketUpdated)
+                            {
+                                Response.Cookies.Append(_generalSettings.GetSBLCookieName, userAuthentication.EncryptedTicket);
```

Comment:
Will tihs simply add a secondary cookie with the same name or overwrite the original one?

Reply:
It will override the existing one